contents/blog/real-user-monitoring.md

```diff
@@ -0,0 +1,114 @@
+
+Because of this, it is a closer representation of reality. It identifies real issues, largely broken down into two classes:
+
+- Performance: loading, page, and query speed
+- Usability: errors, confusion, bugs
+
+![Types](../images/blog/real-user-monitoring/classes.png)
+
+The ideal real user monitoring system identifies these problems, provides details, supports planning of fixes, and ultimately, supports creating a better user experience.
+
+## Why do real user monitoring?
+
+Real user monitoring is important because it helps you understand your current user experience, discover ways to improve it, and manage the risk of future issues.
+
+![Why](../images/blog/real-user-monitoring/why.png)
+
+### 1. Understand current experience
+
+The first reason to do real user monitoring is to ensure your product performs well and provides a good user experience. This means it loads fast, is issue-free, and enables users to accomplish their goals efficiently.
+
+To make this happen, you must have visibility into what the current user experience is like.This means understanding query performance, core web vitals, front and backend error rates, unclear UI, rageclicks, and more. These are signs of a negative user experience to be fixed.
+
+### 2. Improve experience
+
+There are always endless potential improvements to your product. To get the largest impact on what you build, real user monitoring provides metrics to prioritize these improvements. For example:
+
+- A slow query might not matter for a free user but might be a top priority for a large potential customer in a trial stage.
+- Multiple users complain about slow queries, but you don’t know which ones or how slow they are without tracking.
+- A user complains a feature "sucks." This feedback by itself is unclear. It could mean slow page speed, poor design, a bug, or a lack of knowledge.
+
+Real user monitoring helps you get the information to understand and prioritize the issues. It also provides details on how to solve those issues; for example, session replays being bug recreation steps or query metrics showing which ones are slow.
+
+### 3. Manage risk
+
+The third and final reason to do real user monitoring is to manage risk. It supports this by uncovering potential risks, providing details for planning, and being part of the solution. 
+
+By knowing error rates, page and query speeds, and product usage, teams are able to monitor the the health of the product and prevent incidents. These metrics also enables companies to make promises about their experience quality, and back those promises with data. 
+
+For large companies, this is critical. These promises are often solidified in a service-level agreement, or SLA, which is a contract defining the expected service levels an application must provide (such as uptime and performance).
+
+## How to setup real user monitoring with PostHog
+
+Setup happens over three stages: capture, analyze, and action. PostHog has the tools to do all three.
```

Comment:
Assuming this is primarily an SEO article, we should add a little bit more explaining what PostHog is here.